api/urls.py

```diff
@@ -40,6 +40,7 @@
     url(r'rss/?', views.CreativeWorksRSS(), name='rss'),
     url(r'atom/?', views.CreativeWorksAtom(), name='atom'),
     url(r'userinfo/?', views.ShareUserView.as_view(), name='userinfo'),
+    url(r'v1data/?', views.V1DataView.as_view(), name='v1data'),
```

Comment:
Could we map this to the [existing url}(https://osf.io/api/v1/share/data/help/#!/SHARE/post_share_data)?
I think it is `/api/v1/share/data` or something similar.